migrate.py

```diff
@@ -1008,6 +1008,10 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
             }
 
             for plugin_type in spec[namespace][collection].keys():
```

Comment:
Use `.items()` instead.
```suggestion
             for plugin_type, plugins in spec[namespace][collection].items():
```